File: testing.java
import java.util.Random;

public class testing{

    public static void main(String[] args) {
        
        System.out.println("Hello World!");

        Random rand = new Random();

        //This is a comment

        /**
         * Big Comment
         * 
         * 
         * 
         * See told you it was big
         * (That's what she said)
         */

        for(int i = 0; i < 10; i++){

            int randnum = rand.nextInt(0, 10);

            System.out.println("Random Number: " + randnum);

        }

    }

}

File: testing.py
import random

class testing:

    def main():

        print("Hello World!")

        # This is a comment

        """
        Big Comment




        See told you it was big
        (That's what she said)
        """

        for i in range(0, 10):

            rand = random.randint(0, 10)

            print(f"Random Number: {rand}")

if __name__ == '__main__':
    testing.main()